### ISSUE TITLE
app setup issues

### ISSUE THREAD

**Issue by iamavishkar** [twilio/twilio-conversations-demo-react#124]
Hey there!

After cloning when i am installing node modules, it keep saying this to me:

`Module not found: Error: Can't resolve '@twilio-paste/icons/esm/ProductSwitcherIcon' in 'D:\csdev\codesymphony\twilio-conversations-demo-react\node_modules\@twilio-paste\product-switcher\dist'`

However, somehow I fixed this issue myself but after that I am not able to "create a conversation" as the modal keep saying that 
`Cannot read properties of undefined (reading 'then')`

Image has been attached for reference.

![tilio-github-1](https://github.com/twilio/twilio-conversations-demo-react/assets/93832078/b2674a4c-6ab3-42ca-a5d4-03b8c08a5743)

Thank you for you help

**Comment by berkus** [comment]
You seem to be offline - do you actually have a proper token and a connection up?

**Comment by makzpatel** [comment]
@berkus Yes we can login properly with the token. And the connection is also stable.

**Comment by Khyteang** [comment]
Hi @iamavishkar, do you mind sharing what you did to solve the issue
`Module not found: Error: Can't resolve '@twilio-paste/icons/esm/ProductSwitcherIcon' in 'D:\csdev\codesymphony\twilio-conversations-demo-react\node_modules\@twilio-paste\product-switcher\dist'`? Appreciate your help.

**Comment by camr1993** [comment]
I'm also getting `Module not found: Error: Can't resolve '@twilio-paste/icons/esm/ProductSwitcherIcon'`

**Comment by camr1993** [comment]
[this PR](https://github.com/twilio/twilio-conversations-demo-react/pull/125) fixed the issue for me

**Comment by berkus** [comment]
> @berkus Yes we can login properly with the token. And the connection is also stable.

<img width="148" alt="image" src="https://github.com/twilio/twilio-conversations-demo-react/assets/80306/05d26a5b-ee28-4970-b021-005c5821d4b4">

It will not work when Offline.

**Comment by janainadcacaca** [comment]
I have the same problem, I did the setup on my local machine and on Vercel and both have the same problem. It gives an error when configuring Firebase, but the documentation shows that it is optional. It shouldn't give the error.

If you test without adding the firebase configuration, you will realize that the problem exists.

**Comment by equintas** [comment]
Same problem here. Please fix or explain what is wrong with [anainadcacaca](https://github.com/janainadcacaca) and me.

**Comment by agrukhal** [comment]
Closed as there was no follow-up. Should there be more concerns, please, open a new issue
